app/assets/javascripts/usa_map.js

```diff
@@ -19,6 +19,7 @@ $(document).ready(function() {
 
     // as written, the function(error, us) callback won't fire until the d3.json finishes
     d3.json("json/usa_map.json", function(error, us) {
+      console.log(error)
       g.selectAll("path")
```

Comment:
Does this print out an error is one exists? or does it pring all the time? Was this for debugging? or error handling?